templates/web/oxfordshire/reports/_status_filter_options.html

```diff
@@ -0,0 +1,10 @@
+    [% IF c.user.is_superuser OR c.user.from_body %]
+      data-none="[% loc('All unresolved') %]"
+      data-extra="[% loc('All unresolved') %]"
+      data-all-options='["confirmed","investigating","action scheduled","fixed - council","not responsible","duplicate","unable to fix"]'
```

Comment:
Feels like with the above changes to not hardcode it in base, this wouldn't be necessary as it would be the right list already. So this template would only need to add the extra option, plus change data-none?